client/my-sites/promote-post-i2/components/search-bar/index.tsx

```diff
@@ -38,11 +38,21 @@ const SORT_OPTIONS_RECENTLY_UPDATED = 'modified';
 const SORT_OPTIONS_MOST_LIKED = 'like_count';
 const SORT_OPTIONS_MOST_COMMENTED = 'comment_count';
 const SORT_OPTIONS_MOST_VIEWED = 'monthly_view_count';
+const SORT_OPTIONS_BUDGET_HIGH_LOW = 'budget|desc';
+const SORT_OPTIONS_BUDGET_LOW_HIGH = 'budget|asc';
+const SORT_OPTIONS_IMPRESSIONS_HIGH_LOW = 'impressions|desc';
+const SORT_OPTIONS_IMPRESSIONS_LOW_HIGH = 'impressions|asc';
```

Comment:
these will be:
`impressions_total`  and `budget_cents` to respect the contract we have with the campaign/search controller

Reply:
hi @sbarbosa  I fixed this :)